internal/upload/awsupload/awsupload.go

```diff
@@ -243,6 +243,23 @@ func (a *AWS) Register(name, bucket, key string, shareWith []string, rpmArch str
 
 	log.Printf("[AWS] 🎉 AMI registered: %s", *registerOutput.ImageId)
 
+	// Tag the image with the image name.
```

Comment:
What's the relation to the `Name` specified when importing the image? Is the difference just what is used in the UI?

Reply:
Right, the `Name` tag key is a special one and it's used in the UI in AWS when you display an AMI.